cmd/helm/repo.go

```diff
@@ -47,15 +47,22 @@ func runRepoAdd(cmd *cobra.Command, args []string) error {
 	}
 	name, url := args[0], args[1]
 
+	if err := addRepository(name, url); err != nil {
+		return err
+	}
+	fmt.Println(name + " has been added to your repositories")
+	return nil
+}
+
+func addRepository(name, url string) error {
 	if err := downloadIndexFile(name, url); err != nil {
-		return errors.New("Oops! Looks like " + url + " is not a valid chart repository or cannot be reached\n")
+		return errors.New("Looks like " + url + " is not a valid chart repository or cannot be reached\n")
```

Comment:
so when we defined the cobra command, we declared a `RunE` field instead of a `Run` field, so whenever an error gets returned from the method, it will be shown as output on the command line. 
This output is from this branch:
```
helm [add-default-repo]$ bin/helm repo add michelle http://michellecharts.com
Error: Looks like http://michellecharts.com is not a valid chart repository or cannot be reached

Usage:
  helm repo add [flags] [NAME] [URL]

Global Flags:
      --home string                    location of your Helm config (default "$HOME/.helm")
      --log-flush-frequency duration   Maximum number of seconds between log flushes (default 5s)
  -v, --verbose                        enable verbose output

``` 
@adamreese